ocs_ci/templates/app-pods/awscli.yaml

```diff
@@ -6,7 +6,9 @@ metadata:
 spec:
   containers:
     - name: awscli
-      # TODO: Replace the image with an official one once Amazon  publishes theirs
-      image: mikesir87/aws-cli:1.16.220
+      image: amazon/aws-cli:2.0.13
```

Comment:
Shouldn't we use `latest` image to test that we are in sync with the most recent aws-cli?
I am not sure about this - maybe this is ok if it's stable.

Reply:
That's a great point. I am honestly not entirely sure. 
Using a specific tag promises us stability, and assures us that everything will work as expected. It might limit us from things that can be added in future revisions, but at the moment, we don't seem to need those. That's why I chose to go with the most futureproof way - and if we'll need to use a feature added in a specific CLI version, we can always update the YAML and verify that everything is working well.

What do you think?

Reply:
Also, the description of the image clearly states that [no backwards compatibility is promised when using the `latest` tag](https://hub.docker.com/r/amazon/aws-cli#notes-on-interfaces-and-backwards-compatibility-of-docker-image)

Reply:
I will leave this up to you or if anyone else wants to decide this.
> Also, the description of the image states that [no backwards compatibility is promised when using the `latest` tag](https://hub.docker.com/r/amazon/aws-cli#notes-on-interfaces-and-backwards-compatibility-of-docker-image).

If we would use `latest` tag then we wouldn't miss changes in aws-cli related to our use cases. We would see that tests that were passing are not passing because of some change in aws-cli.

Reply:
You're right; I'll have some more discussions regarding this. Thanks for bringing it up!

Reply:
@MeridianExplorer should we wait with merging until we reach a final decision?

Reply:
@ebenahar 
I think it'd be best to merge the PR ASAP, and change it down the road in case we choose to.

Reply:
@MeridianExplorer please file an issue to track filip's comment on using latest tag for the image.